introduction_to_amazon_algorithms/forecasting_services_comparison/forecast_example.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<p>Important to have a Cleanup section before Next Steps/Conclusion, deleting the predictor or any other resources that will keep incurring costs. Applies to both notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>